### PR TITLE
Controllers and models

### PR DESCRIPTION
**Pull request by orianacolmenares**
- Create controller 
**Trello card:**  https://trello.com/c/UBZLs9Az/17-crear-un-controller-y-una-vista-de-bienvenida

- Create book model 
**Trello card:** https://trello.com/c/ORS46VGO/14-crear-modelo-de-book

- Create book repository
**Trello card:** https://trello.com/c/nT03LrSa/15-crear-un-repositorio-de-books

### CODE REVIEW

File: src/main/java/wolox/training/models/Book.java
Comment (IgnOliveto) [inline src/main/java/wolox/training/models/Book.java:13]: mmm i think you miss the getter for the id and we need a  long id attribute .

**Review by nicolasmederoc — APPROVED** [review]
Seems good to me :)

We need to wait for @IgnOliveto 's approval :rocket:

**Review by IgnOliveto — APPROVED** [review]

**Pull request merged**
